packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -71,7 +71,7 @@ describe("MergeTree.Client", () => {
             const mt = random.engines.mt19937();
             mt.seedWithArray([0xDEADBEEF, 0xFEEDBED, minLength]);
 
-            const clients: TestClient[] = [new TestClient({ blockUpdateMarkers: true })];
```

Comment:
I'd consider going further and removing options argument in Client's constructor.
I do not see it used anywhere.

Reply:
It looks like the `options` argument is passed into the constructor of the `MergeTree` object and it seems there is one place where we pass a defined value:
![image](https://user-images.githubusercontent.com/102998837/162092026-d1dec7de-ac2e-4c1b-80f4-c40c2f97a96f.png)

Did you still want me to remove the argument from the `Client` constructor?

Reply:
I did not see any usage of this.options in Client class. So, it seems like it could be removed, no?

Reply:
lets keep this for now. there are few options still in use